sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16601,7 +16601,7 @@
               },
               "key": "estimatedSpouseIncome",
               "conditional": {
-                "show": true,
+                "show": "true",
```

Comment:
Why did it change?

Reply:
This code has been reverted due to the comment: https://github.com/bcgov/SIMS/pull/2787#pullrequestreview-1860177683 
However, as per our discussion @andrepestana-aot , I have checked and the conditional show and hide is working properly for this component.